.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -456,6 +507,53 @@ jobs:
           sudo chmod 600 ~/.ssh/private.key
           rsync --progress -avz -e "ssh -i ~/.ssh/private.key -o StrictHostKeyChecking=no" $GITHUB_WORKSPACE/* "$USER@$HOST:/OpenMSInstaller/${folder}"
 
+      - name: create RELEASE_TEXT
+        if: inputs.do_release
+        shell: bash
+        run: |
+            #TODO Handle changelog_params
+            ## Header
+            cat > releaseTextHeader.txt << EOF
+            Dear OpenMS-Users,
+
+            we are proud to announce the release of OpenMS $version. Grab it
+            <a href="https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/$version">here</a>
+
+            In the following you find all important changes to the previous version:
+
+            EOF
+
+            ## Header
+            cat > releaseTextParamHeader.txt << EOF
+
+            Additionally, the following changes were made to parameters of our TOPP tools:
+
+            EOF
+
+            ##Footer
+            cat > releaseTextFooter.txt << EOF
+
+            Best regards,
+            The OpenMS-Developers
+
+            EOF
+            RELEASE_TEXT=$(cat releaseTextHeader.txt $GITHUB_WORKSPACE/changelog.txt releaseTextFooter.txt)
+            RELEASE_TEXT_ESCAPED_QUOTES="${RELEASE_TEXT//\"/\\\"}"
+            cat releaseTextHeader.txt $GITHUB_WORKSPACE/changelog.txt releaseTextFooter.txt > RELEASE_TEXT_GH.md
+            # cat RELEASE_TEXT_GH.md | jq -R -s '.' > RELEASE_TEXT_GH.json
+
+      - name: run GHA release action.
+        if: inputs.do_release
+        uses: softprops/action-gh-release@v1
+        with: 
+          body_path: RELEASE_TEXT_GH.md
+          name: ${{ steps.extract_branch.outputs.RUN_NAME }}
+          files:
+            ${{ github.workspace }}/*.exe
+            ${{ github.workspace }}/*.deb
+            ${{ github.workspace }}/*.pkg
+            # TODO add src
```

Comment:
In theory a src gz is always available at the release page, separately from the artifacts but we are currently adding a modfied src.gz with docs, with checked out contrib (and Thirdparty?) and without .git index.

Reply:
Okay I'm fine with leaving it out of here then.

Reply:
Ah, okay. I can look into that after I get the current batch of changes to work. I have to say, I hadn't realized just how much patience it takes to get all the CI to work. Props to you for all of your work on it over the years.